noop.go

```diff
@@ -0,0 +1,60 @@
+package log
```

Comment:
Instead of using a Noop Logger why not use a existing logger like Logrus with the default configs?

Reply:
With the current project structure, this would cause a cyclic dependency problem since the other implementations depend on the `log` package.

Reply:
As pointed on by @dubonzi and explained in the PR description, it cannot be done due to a cyclic dependency. Unfortunately, this is not only the project structure fault but a consequence of Go's invariant type system. See the description for more details.